epytope/Core/Result.py

```diff
@@ -27,8 +33,7 @@ def filter_result(self, expressions):
         """
         Filter result based on a list of expressions
 
-        :param list((str,comparator,float)) expressions: A list of triples consisting of (method_name, comparator,
-                                                         threshold)
+        :param list((str,comparator,float)) expressions: A list of triples consisting of (method_name, comparator,threshold)
```

Comment:
Just as a little comment: it would be nice to make consistent use of whitespaces in the list of triples (either one whitespace after each comma like in line 86 or none at all)